### DISCUSSION TITLE
Allow to choose which Firefox keyboard shortcuts/hotkeys is reconfigured instead of disabling every Firefox keyboard shortcuts

### DISCUSSION THREAD

**Discussion by DivineMK** [Floorp-Projects/Floorp#1045] [Feature Request]
Currently I know of floorp.custom.shortcutkeysAndActions.remove.fx.actions for disabling Firefox default keyboard shortcuts/hotkeys but I believe a lot of common hotkeys are preferably left unchanged (such as Ctrl+w for close current tab, Ctrl+t for new tab...).
Hence, I propose to allow users to select hotkeys they want to override and keep the rest unchanged instead. Related to #995.

**Comment by MoonlessForest** [comment]
Personally, I'd also like to be able to simply enable or disable each one of them individually with a toggle.
